docs/game/cheat_sheet.md

```diff
@@ -12,37 +12,42 @@ enjoyable, entertaining and informative show. Winning isn't everything.
 Manners
 -------
 
-Some general manners guidelines for panel games, and specifically Just A Minute
+Panel games, much like the greatness of Great Britain, are founded on
+good manners. Here are some guidelines that make Just A Minute flow smoothly.
```

Comment:
That's a definite improvement. 

My thinking here is that this is also a cheat-sheet for presenting as part of a panel, not just playing the game. If we can't speak in turn and present a clear narrative to the audience then it doesn't matter who wins, it's a really messy segment.